frontend/src/APIClients/AuthAPIClient.ts

```diff
@@ -0,0 +1,62 @@
+import baseAPIClient from "./BaseAPIClient";
+import USER_ACCESS_TOKEN_KEY from "../constants/AuthConstants";
+
+const login = async (email: string, password: string): Promise<boolean> => {
+  try {
+    const { data } = await baseAPIClient.post(
+      "/auth/login",
+      { email, password },
+      { withCredentials: true },
+    );
+    localStorage.setItem(USER_ACCESS_TOKEN_KEY, data.accessToken);
```

Comment:
As previously discussed, the access token is short-lived so we're not too concerned with storing in local storage